products/bash/release-mq.sh

```diff
@@ -46,13 +50,18 @@ while getopts "n:r:i:q:t" opt; do
       ;;
     q ) qm_name="$OPTARG"
       ;;
-    t ) tracing=true
+    t ) tracing_namespace="$OPTARG"
       ;;
     \? ) usage; exit
       ;;
   esac
 done
 
+
+# tracing false if tracing namespace is empty
+[ -z $tracing_namespace ] && tracing_enabled="false"
```

Comment:
is tracing_enabled ever actually false here?  I can't see how you'd run this script without tracing on.